components/brave_wallet_ui/components/desktop/lock-screen/index.tsx

```diff
@@ -99,6 +102,39 @@ export const LockScreen = () => {
 
   const isAndroid = loadTimeData.getBoolean('isAndroid') || false
 
+  if (isAndroid) {
+    // Computed
+    let lastPress = 0
+
+    // Methods
+    const onDoubleTap = () => {
+      const time = new Date().getTime()
+      const delta = time - lastPress
+      if (delta < 400) {
+        getWalletPageApiProxy().pageHandler.unlockWalletUI()
+      }
+      lastPress = time
+    }
```

Comment:
lets use the event detail to check if this was a double-click to better handle user accessibility: https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event#usage_notes